server/src/main/scala/org/http4s/server/websocket/package.scala

```diff
@@ -40,13 +40,13 @@ package object websocket {
     * @param status The status code to return to a client making a non-websocket HTTP request to this route
     */
   def WS[F[_]](
-      read: Stream[F, WebSocketFrame],
-      write: Sink[F, WebSocketFrame],
+      send: Stream[F, WebSocketFrame],
```

Comment:
@aeons I'm unsure how to handle the deprecation here

Reply:
I don't think we can in this case.  It'll break for people using named arguments, but at least the types are unique.  I think it'll be okay.

Reply:
:+1: to this as well. It should be a simple fix for users.